.github/workflows/releaser.yml

```diff
@@ -108,7 +105,7 @@ jobs:
       actions: read # To read the workflow path.
       id-token: write # To sign the provenance.
       contents: write # To add assets to a release.
-    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.9.0
```

Comment:
@JAORMX @jhrozek - I recall we should exclude some of these (one had a problem with the digest reference)

Reply:
@rdimitrov you are right, https://github.com/stacklok/frizbee/blob/main/.frizbee.yml#L5

Reply:
This is configurable via minder though, it's just about setting up the right `excludes` in there.

Reply:
until it is configured we should hold off merging this one then 👍

Reply:
This is our configuration:
```
  - type: actions_check_pinned_tags
    def:
      exclude:
        - slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@1.9.0
```
It also marks the frizbee repo as succeeded, so currently I'm unsure what's going on

Reply:
Also strange that the CLA got triggered, I remember we added the bot when we did the first enrolment. cc: @dussab

Reply:
@jhrozek aha! the tag doesn't match. The configuration uses `1.9.0` while we have configured `v1.9.0` in the action.

Reply:
@jhrozek the right tag seems to be `v1.9.0`

Reply:
hawk-eye-ozz in action! 🦅 😃

Reply:
This was fixed by modifying the profile. Thanks for the sharp eyes!